### ISSUE TITLE
redemrp

### ISSUE THREAD

**Issue by LOTW1869** [unknownghostz6/westerntheme-mdt#1]
please support for old redemrp and redemrp 2023

**Comment by unknownghostz6** [comment]
ill setup a test server later on redemrp today or tomorrow and try and get it done for those frameworks within the next couple of days but it will be my newest version of this that will be made for it
https://github.com/unknownghostz6/ghost_mdt

itll be liike this one but ill get it converted for the redemrp frameworks within next few days

**Comment by LOTW1869** [comment]
yes, u can try for old redemrp and redemrp 2023, its nice framework 

this is old redemrp
https://docs.kanersps.pw/docs/redem/classes/player/

and this is for redemrp2023
https://sinatra.gitbook.io/redemrp/
